lib/streamlit/elements/dialog_decorator.py

```diff
@@ -94,56 +94,80 @@ def dialog_decorator(title: F | None, *, width: DialogWidth = "small") -> F:
 def dialog_decorator(
     title: F | None | str = "", *, width: DialogWidth = "small"
 ) -> F | Callable[[F], F]:
-    r"""Decorate a function to mark it as a Streamlit dialog. When the decorated function is called, a dialog element is inserted with the function's body as the content.
+    """Decorator to turn a function into a dialog function which creates a modal overlay when called.
 
-    The decorated function can hold multiple elements which are rendered inside of a modal when the decorated function is called.
-    The decorated function is using `st.experimental_fragment`, which means that interacting with elements inside of the dialog will
-    only re-run the dialog function.
+    When you call a function decorated with ``@st.experimental_dialog``, a
+    modal element is inserted into your app. Streamlit element commands called
```

Comment:
"modal element" -> "modal dialog" (or "dialog element" but I think the use of "element" is a tiny bit weird here, since it isn't something that goes into your normal app flow)

Reply:
maybe better is something with `container`, e.g. `modal window container`, `modal container`, `dialog container`

Reply:
I thought about using the word container, but ended up thinking it's better to avoid since this isn't a DeltaGenerator object you can write `with`...